pkg/resources/sts/sts.go

```diff
@@ -13,19 +13,21 @@ import (
 )
 
 const (
-	clusterCloudCredentialName  = "cluster"
+	ClusterCloudCredentialName  = "cluster"
 	RoleArnParameterName        = "sts-role-arn"
 	RoleSessionName             = "Red-Hat-cloud-resources-operator"
 	CredsSecretName             = "sts-credentials"
 	CredsSecretRoleARNKeyName   = "role_arn"
 	CredsSecretTokenPathKeyName = "web_identity_token_file"
 	CredsRoleEnvKey             = "ROLE_ARN"
 	CredsTokenPathEnvKey        = "TOKEN_PATH"
+	CredsTmpS3AccessKeyId       = "s3-access-key-id"
+	CredsTmpS3SecretAccessKey   = "s3-secret-access-key"
```

Comment:
```suggestion
	CredsS3AccessKeyId       = "s3-access-key-id"
	CredsS3SecretAccessKey   = "s3-secret-access-key"
```
Aligning to remove `Tmp` from name :thinking:

Reply:
Done. Thank you